sherpa/astro/ui/serialize.py

```diff
@@ -1487,14 +1645,21 @@ def save_all(state: SessionType, fh: TextIO | None = None) -> None:
      3. some settings and values may not be recorded.
 
      .. versionchanged:: 4.18.0
-        Handling of PHA2 data sets has been improved.
+        Handling of PHA data has been improved, and the output now
+        defaults to not including automatically-loaded ancillary files
+        (such as background and responses). Set the ``auto_load``
+        flag to False to add these commands back to the save file (and
+        match previous versions of Sherpa).
 
```

Comment:
Could we add the note how to use the saved file? As a 'sherpa script' - but I don't like this description. 
I used 'run -i test_all' for 'test_all' file saved with 'save_all'

Reply:
I added a small section to the sherpa.astro.ui.utils.save_all command (since this is what CIAO users will see) to

https://github.com/sherpa/sherpa/pull/2382/commits/e3a3f020607b53e40ab2ddce339eebacd684b20e